pegasus/sites.v3/code.org/views/stats_homepage.haml

```diff
@@ -0,0 +1,21 @@
+- student_accounts_percent = "20%"
+- female_students = "9M"
+- teacher_users = "600K"
+- world_students_hoc_percent = "10%"
+- states_policy_change = "20"
+
+- top_statistic_number = student_accounts_percent
+- top_statistic_phrase = "of US students have accounts on Code.org"
+
+- statistics = { female_students => "of our students are female", teacher_users => "teachers use Code.org", world_students_hoc_percent => "of the world's students have tried the Hour of Code", states_policy_change => "states changed policy to support computer science" }
```

Comment:
Would be good to break this up into multiple lines.  Could also use `:ruby` for this entire block.